src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -861,118 +965,259 @@ private ArrayList<BrAPIGermplasm> getGermplasmByAccessionNumber(
-        Map<String, PendingImportObject<BrAPIStudy>> studyByNameNoScope = new HashMap<>();
-        if( this.trialByNameNoScope.size()!=1){
-            return studyByNameNoScope;
+    private Map<String, PendingImportObject<BrAPITrial>> initializeTrialByNameNoScope(Program program, List<ExperimentObservation> experimentImportRows) {
+        Map<String, PendingImportObject<BrAPITrial>> trialByName = new HashMap<>();
+        String programKey = program.getKey();
+
+        initializeTrialsForExistingObservationUnits(program, experimentImportRows, trialByName);
+
+        List<String> uniqueTrialNames = experimentImportRows.stream()
+                                                            .filter(row -> StringUtils.isBlank(row.getObsUnitID()))
+                                                            .map(experimentImport -> Utilities.appendProgramKey(experimentImport.getExpTitle(), programKey))
+                                                            .distinct()
+                                                            .collect(Collectors.toList());
+        try {
+            String trialRefSource = String.format("%s/%s", BRAPI_REFERENCE_SOURCE, ExternalReferenceSource.TRIALS.getName());
+            brapiTrialDAO.getTrialsByName(uniqueTrialNames, program)
+                         .forEach(existingTrial -> processAndCacheTrial(existingTrial, program, trialRefSource, trialByName));
+        } catch (ApiException e) {
+            log.error("Error fetching trials: " + Utilities.generateApiExceptionLogMessage(e), e);
+            throw new InternalServerException(e.toString(), e);
         }
-        ExperimentObservation experimentObservation = experimentImportRows.get(0);
 
-        PendingImportObject<BrAPITrial> trial = this.trialByNameNoScope.get(experimentObservation.getExpTitle());
+        return trialByName;
+    }
 
-        UUID experimentId = trial.getId();
+    private Map<String, PendingImportObject<BrAPIStudy>> initializeStudyByNameNoScope(Program program, List<ExperimentObservation> experimentImportRows) {
+        Map<String, PendingImportObject<BrAPIStudy>> studyByName = new HashMap<>();
+        if (this.trialByNameNoScope.size() != 1) {
+            return studyByName;
+        }
 
         List<BrAPIStudy> existingStudies;
+        Optional<String> expTitle = experimentImportRows.stream()
+                                                        .filter(row -> StringUtils.isBlank(row.getObsUnitID()))
+                                                        .map(ExperimentObservation::getExpTitle)
+                                                        .findFirst();
+        if(expTitle.isEmpty()) {
+            expTitle = trialByNameNoScope.keySet().stream().findFirst();
+        }
+        PendingImportObject<BrAPITrial> trial = this.trialByNameNoScope.get(expTitle.get());
```

Comment:
I think this is ok because at the top of this method there is a check to see if the `trialByNameNoScope` has exactly 1 element, and if not, return.  So this code won't get hit unless that map is populated